app/screens/ProductList/ProductList.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import { Text, View, Image, TouchableHighlight } from 'react-native';
+
+import { iconImages } from './../../images';
+
+import { styles } from './styles';
+
+class ProductList extends Component {
+    render() {
+        return (
+            <View style={styles.container}>
+                <Text style={styles.title}>Products</Text>
+                <View style={styles.productItem}>
+                    <View style={styles.productInfo}>
```

Comment:
const data  = [
 { 
    title: 'Product 1',
    image: iconImages.emptyJamJar
},
 { 
    title: 'Product 2',
    image: iconImages.emptyJamJar
},
 { 
    title: 'Product 3',
    image: iconImages.emptyJamJar
}
]
....
renderItems = () => {
return data.map(({ title, image }) => (
  <View style={styles.productItem}>
                    <View style={styles.productInfo}>
                        <Image
                            source={image}
                            style={styles.productImage}/>
                        <Text style={styles.productTitle}>{title}</Text>
                    </View>
                    <TouchableHighlight
                        onPress={() => alert('OK')}
                        underlayColor='#263852'
                        style={styles.button}>
                        <Text style={styles.buttonText}>login</Text>
                    </TouchableHighlight>
                </View>
))
}

.....
render() {
        <View style={styles.container}>
                <Text style={styles.title}>Products</Text>
                   {this.renderItems()}
       </View>
}